src/elmUtils.ts

```diff
@@ -64,4 +64,4 @@ export function getIndicesOf(searchStr : string, str : string) : number[] {
   return indices;
 }
 
-export const pluginPath = (isWindows ? process.env["USERPROFILE"] : process.env["HOME"]) + path.sep + '.vscode' + path.sep + 'extensions' + path.sep + 'sbrink.elm'
\ No newline at end of file
+export const pluginPath = (isWindows ? process.env["USERPROFILE"] : process.env["HOME"]) + path.sep + '.vscode' + path.sep + 'extensions' + path.sep + 'sbrink.elm-0.3.0'
```

Comment:
This is gonna be a pain to keep updated and won't work when testing/using the extension from another location. We can use this instead:

``` typescript
export const pluginPath = path.join(__dirname, "../..")
```